providers/os/resources/npm/package_json.go

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package npm
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"regexp"
+	"strings"
+
+	"go.mondoo.com/cnquery/v10/providers-sdk/v1/upstream/mvd"
+)
+
+// packageJson allows parsing the package json file
+type packageJson struct {
+	Name            string                `json:"name"`
+	Description     string                `json:"description"`
+	Version         string                `json:"version"`
+	Private         bool                  `json:"private"`
+	Homepage        string                `json:"homepage"`
+	License         *packageJsonLicense   `json:"license"`
+	Author          *packageJsonPeople    `json:"author"`
+	Contributors    []packageJsonPeople   `json:"contributors"`
+	Dependencies    map[string]string     `jsonn:"dependencies"`
+	DevDependencies map[string]string     `jsonn:"devDependencies"`
+	Repository      packageJsonRepository `json:"repository"`
+	Engines         map[string]string     `jsonn:"engines"`
+	CPU             []string              `json:"cpu"`
+	OS              []string              `json:"os"`
+}
+
+// packageJsonPeople represents the author of the package
+// https://docs.npmjs.com/cli/v10/configuring-npm/package-json#people-fields-author-contributors
+type packageJsonPeople struct {
```

Comment:
```suggestion
type packageJsonAuthor struct {
```

Reply:
We should not do this since the same struct is used for contributors as well. npm docs itself call this people so lets stick with that.